puphpet/config.yaml.template

```diff
@@ -79,25 +100,31 @@ apache:
     settings:
         user: www-data
         group: www-data
-        default_vhost: true
+        default_vhost: false
```

Comment:
Why is that bad? I think Rick setup the nginx host to respond to anything on the specified IP address.